src/main/java/com/angelbroking/smartapi/smartstream/ticker/SmartStreamTicker.java

```diff
@@ -27,7 +26,10 @@
 @Slf4j
 public class SmartStreamTicker {
 
-	private static final int PING_INTERVAL = 10000; // 10 seconds
+	private static int PING_INTERVAL_IN_SECONDS = 10000; // 10 seconds

```

Comment:
These are milliseconds not seconds. please change variable names.

Reply:
Variables names are changed accordingly. Please check.